CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
 # MyUW hrs-portlets change log
 
+### Unreleased (2.2?)
 
-### Unreleased
+New features
+
++ Add mapping for new `UW_DYN_AM_PUNCH_TIME` HRS role. Split out from the
+  existing `ROLE_VIEW_ABSENCE_HISTORIES` role a role specific to creating,
+  editing, and canceling one's own absence requests
+  (`ROLE_ENTER_EDIT_CANCEL_OWN_ABSENCES`). Grant only the absence history
+  viewing (and not the entering, editing, canceling one's own absences) to the
+  new `UW_DYN_AM_PUNCH_TIME` HRS role. Grant
+  `ROLE_ENTER_EDIT_CANCEL_OWN_ABSENCES` to existing `UW_DYN_AM_EMPLOYEE` role so
+  that its access is unchanged. ( #121 )
```

Comment:
I think this description is a bit tough to decipher.

Reply:
Took a shot at clarifying, that help?